### PR TITLE
[#1559] New associations in service

### LINKED ISSUE #1559: Add new associations in service model
Add new associations:
- funding_body
- funding_program

### PR DESCRIPTION
**Pull request by goreck888**
Add new associations mentioned as API vocabularies:
- FundingBody
- FundingProgram

Fixes #1559

### CODE REVIEW

**Comment by agpul** [comment]
I can't reach this PR, @wziajka can you check?

File: app/models/funding_body.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class FundingBody < Vocabulary
+  has_many :service_vocabularies, as: :vocabulary, source: "FundingBody"
```
[the remark this answers is not in the document]
Comment (goreck888) [reply app/models/funding_body.rb:4]: You are right. Done.

File: app/models/service_vocabulary.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ServiceVocabulary < ApplicationRecord
+  belongs_to :service
+  belongs_to :vocabulary, polymorphic: true
```
[the remark this answers is not in the document]
Comment (goreck888) [reply app/models/service_vocabulary.rb:5]: It has to be polymorphic. We have one table for different child objects. When you remove `polymorphic: true` option, it try to find different associations table.

File: .rake_tasks~ [hunk 3]
```diff
@@ -0,0 +1,95 @@
+about
```
Comment (martaswiatkowska) [inline .rake_tasks~:1]: What's that?

Comment (goreck888) [reply .rake_tasks~:1]: Sorry, this file creates always when something goes wrong with rake task. I delete it now.

**Comment by goreck888** [comment]
> Add also loading vocabularies to dev:prime

Done.

File: lib/tasks/dev.rake [hunk 4]
```diff
@@ -5,6 +5,7 @@ namespace :dev do
   task prime: "db:setup" do
     yaml_hash = YAML.load_file("db/data.yml")
 
+    create_vocabularies
```
Comment (martaswiatkowska) [inline lib/tasks/dev.rake:8]: Add To service.update! some examples of this new attributes

Comment (goreck888) [reply lib/tasks/dev.rake:8]: Done.

**Pull request merged**